internal/confmapprovider/discovery/bundle/bundle.d/receivers/oracledb.discovery.yaml.tmpl

```diff
@@ -0,0 +1,64 @@
+{{ receiver "oracledb" }}:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)oracle"}) and not (command matches "splunk.discovery")
+    host_observer: type == "hostport" and command matches "(?i)oracle" and not (command matches "splunk.discovery")
+    k8s_observer: type == "port" and pod.name matches "(?i)oracle"
+  config:
+    default:
+      endpoint: {{ defaultValue }}
+      username: {{ defaultValue }}
+      password: {{ defaultValue }}
+      service: {{ defaultValue }}
+  status:
+    metrics:
+      successful:
+        - strict: oracledb.cpu_time
+          first_only: true
+          log_record:
+            severity_text: info
+            body: oracledb receiver is working!
+    statements:
+      failed:
```

Comment:
When I test w/ example.com as an endpoint I see

```
{"level":"debug","ts":1695160969.7470543,"msg":"evaluating statement","kind":"receiver","statement":{"Message":"Error scraping metrics","Fields":{"caller":"scraperhelper/scrapercontroller.go:200","error":"error executing select * from v$sysstat: EOF; error executing select status, type, count(*) as VALUE FROM v$session GROUP BY status, type: EOF; error executing select RESOURCE_NAME, CURRENT_UTILIZATION, LIMIT_VALUE, CASE WHEN TRIM(INITIAL_ALLOCATION) LIKE 'UNLIMITED' THEN '-1' ELSE TRIM(INITIAL_ALLOCATION) END as INITIAL_ALLOCATION, CASE WHEN TRIM(LIMIT_VALUE) LIKE 'UNLIMITED' THEN '-1' ELSE TRIM(LIMIT_VALUE) END as LIMIT_VALUE from v$resource_limit: EOF; error executing select TABLESPACE_NAME, BYTES from DBA_DATA_FILES: EOF; error executing select TABLESPACE_NAME, (BLOCK_SIZE*MAX_EXTENTS) AS VALUE FROM DBA_TABLESPACES: EOF","kind":"receiver","name":"oracledb//receiver_creator/docker_observer{endpoint=\"example.com:80\"}/6437e17e49a001183bc3686e4300a17127b4129135cd17ee59a3afa53c3bda99:1521","scraper":"oracledb//receiver_creator/docker_observer{endpoint=\"example.com:80\"}/6437e17e49a001183bc3686e4300a17127b4129135cd17ee59a3afa53c3bda99:1521","stacktrace":"go.opentelemetry.io/collector/receiver/scraperhelper.(*controller).scrapeMetricsAndReport\n\tgo.opentelemetry.io/collector/receiver@v0.85.0/scraperhelper/scrapercontroller.go:200\ngo.opentelemetry.io/collector/receiver/scraperhelper.(*controller).startScraping.func1\n\tgo.opentelemetry.io/collector/receiver@v0.85.0/scraperhelper/scrapercontroller.go:176"},"Level":"error","Time":"2023-09-19T22:02:49.746966606Z","LoggerName":"","Caller":{"Defined":true,"PC":42957028,"File":"go.opentelemetry.io/collector/receiver@v0.85.0/scraperhelper/scrapercontroller.go","Line":200,"Function":"go.opentelemetry.io/collector/receiver/scraperhelper.(*controller).scrapeMetricsAndReport"},"Stack":"go.opentelemetry.io/collector/receiver/scraperhelper.(*controller).scrapeMetricsAndReport\n\tgo.opentelemetry.io/collector/receiver@v0.85.0/scraperhelper/scrapercontroller.go:200\ngo.opentelemetry.io/collector/receiver/scraperhelper.(*controller).startScraping.func1\n\tgo.opentelemetry.io/collector/receiver@v0.85.0/scraperhelper/scrapercontroller.go:176"}}
```

Maybe another failed regexp like `error executing select .*: EOF`?

Reply:
I somehow missed putting that one in. Thank you for double checking!